src/main/resources/assets/paths/progressReport.yaml

```diff
@@ -0,0 +1,35 @@
+post:
+  summary: Create a new DAR progress report
+  description: Create a new DAR progress report
+  tags:
+    - Data Access Request
+  parameters:
+    - name: parentReferenceId
+      in: path
+      description: The referenceId of the Data Access Request
+      required: true
+      schema:
+        type: string
+  requestBody:
+    content:
+      required: true
+      application/json:
+        schema:
+          $ref: '../schemas/DataAccessRequest.yaml'
```

Comment:
Posting this via swagger, I get
```
{
  "code": 415,
  "message": "HTTP 415 Unsupported Media Type"
}
```
I believe `content` needs to be under the multipart/form-data section (https://swagger.io/docs/specification/describing-request-body/multipart-requests/)

Reply:
Ok, I was able to post content by moving everything under `multipart/form-data`. I still can't get it to recognize the schema of the dar though.